grails-app/controllers/org/broadinstitute/orsp/api/DataUseController.groovy

```diff
@@ -89,6 +81,18 @@ class DataUseController extends AuthenticatedController {
         }
     }
 
+    def saveSdul() {
+        try {
+            DataUseRestriction restriction = dataUseLetterService.getRestrictionFromParams(request.JSON)
```

Comment:
Follow-up from previous feedback. This method can throw an illegal argument exception which is converted into a 500 below. We should catch that case and return a BadRequest (400) instead.

Reply:
additional catch added